examples/custom-image.json

```diff
@@ -0,0 +1,39 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes"
+    },
+    "masterProfile": {
+      "imageName": "stretch",
+      "imageResourceGroup": "debian",
+      "count": 1,
+      "dnsPrefix": "mydns",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool1",
+        "count": 3,
+        "imageName": "stretch",
+        "imageResourceGroup": "debian",
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet"
+      }
+    ],
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": "mykeyData"
```

Comment:
nit: let's keep these boilerplate non-vals empty strings in the example api model (ditto `dnsPrefix` above and `clientId`/`secret` below)

Reply:
updated